recipe/build_base.sh

```diff
@@ -532,7 +532,10 @@ if [[ ${HOST} =~ .*linux.* ]]; then
   echo "See: https://github.com/conda/conda/issues/6030 for more information."                                   >> ${PREFIX}/shared/python_compiler_compat/README
 fi
 
+# Compile all Python files
+# Ensure `sysconfig`'s data is also compiled
 python -c "import compileall,os;compileall.compile_dir(os.environ['PREFIX'])"
+python -c "import pprint, sysconfig; pprint(sysconfig.get_config_vars())"
```

Comment:
As there is a fair bit of logic above to handle the generation of `sysconfig` data, there may be a better approach than this. Open to suggestions

Reply:
That thread is about a file being changed. Specifically `lib/python3.12/__pycache__/_sysconfigdata__linux_aarch64-linux-gnu.cpython-312.pyc`. Line 537 above already does this, so your changes just doesn't do anything.